docs/docs/learn/concepts/storage/notes.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Understanding Notes
+description: Core knowledge of Notes and how they work
+---
+
+Most prominant blockchain networks don't have privacy at the protocol level. Aztec contracts can define public and private functions, that can read/write public and private state.
+
+For private state we need encryption and techniques to hide information about state changes. For private functions, we need local execution and proof of correct execution.
+
+### Some context
+- public functions and storage work much like other blockchains in terms of having dedicated storage slots
+- Private functions are executed locally with proofs generated for sound execution, and commitments to private variable updates are stored using append-only trees.
+- "Note" types are part of aztec.nr, a framework that facilitates use of Aztec's different storage trees to achieve private variables. Note: there are more uses beyond this.
```

Comment:
```suggestion
- "Note" types are part of Aztec.nr, a framework that facilitates use of Aztec's different storage trees to achieve private variables. Note: there are more uses beyond this.
```

Reply:
I'm not sure we actually need this. Maybe just mention something along the lines of "Notes are abstracted for smart contract developers through the Aztec.nr framework. You can read more about that here (link to other doc in build)

Reply:
That omits the Why of notes, what it seeks to achieve (private variables and more). Also see comments at this section in original hackmd.